README.md

```diff
@@ -185,6 +187,27 @@ OPTIONS
 
 _See code: [src/commands/server/delete.ts](https://github.com/che-incubator/chectl/blob/v0.0.2/src/commands/server/delete.ts)_
 
+## `chectl server:logs`
+
+Retrieve Eclipse Che Logs
+
+```
+USAGE
+  $ chectl server:logs
+
+OPTIONS
+  -h, --help  show CLI help
+
+  -n, --chenamespace=chenamespace
+      [default: che] Kubernetes namespace where Che server is supposed to be deployed
+
+  --deployment-name=deployment-name
+      [default: che] Che deployment name
+
+  -d, --directory=directory
+      [default: ./logs] Directory to store Eclipse Che logs
```

Comment:
Looks like readme isn't regenerated after the latest changes :slightly_smiling_face: 
see `yarn pack`